doc/DataPlaneCodeGeneration/DeveloperDrivenEvolution.md

```diff
@@ -205,3 +210,76 @@ namespace Azure.AI.MetricsAdvisor
     }
 }
 ```
+
+### Implement LRO method
+#### Case: With RequestContent but no return value
+**Generated code before (Generated/DeviceManagementClient.cs):**
+```C#
+namespace Azure.IoT.DeviceUpdate
+{
+    public partial class DeviceManagementClient
+    {
+        public virtual Operation ImportDevices(WaitUntil waitUntil, string action, RequestContent content, RequestContext context = null)
+        {
+            using var scope = ClientDiagnostics.CreateScope("DeviceManagementClient.ImportDevices");
+        }
+    }
+}
+```
+**Improve the LRO method ({Other manual folder}/DeviceManagementClient.cs):**
+```C#
+namespace Azure.IoT.DeviceUpdate
+{
+    public partial class DeviceManagementClient
+    {
+        // The return type is still Operation because no response value
+        public virtual Operation ImportDevices(WaitUntil waitUntil, string action, ImportType type, CancellationToken cancellationToken = default)
+        {
+            // We don't open a new scope here because it is just a wrapper
+            // Convert model to binary content
+            RequestContent requestContent = ImportType.ToRequestContent(type);
+
+            // Call protocol method and return the operation
+            return await ImportDevices(waitUntil, action, requestContent, new RequestContext() { CancellationToken = cancellationToken }).ConfigureAwait(false);
+        }
+    }
+}
+```
+#### Case: Without RequestContent and return value
+There is no model returned and no model as an input. Therefore, don't add a convenience method as a wrapper.
+#### Case: With a return value
+**Generated code before (Generated/DeviceManagementClient.cs):**
+```C#
+namespace Azure.IoT.DeviceUpdate
+{
+    public partial class DeviceManagementClient
+    {
+        public virtual Operation<Response> ImportDevices(WaitUntil waitUntil, string action, RequestContent content, RequestContext context = null)
```

Comment:
`Operation<Response>` is not a valid return type -- it would be `Operation<BinaryData>` for a protocol method.